RatLib/ScreenScale.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RatLib
+{
+	// Class used to communicate between RatScanner and RatRazor the screens available and their scales
+	public class ScreenScale
```

Comment:
A ScreenScale is essentially a Windows display with the scaling factor calculated for it. This is created by ScreenInfo.cs in RatScanner, but this class is used in both RatScanner and RatRazor